src/mscorlib/src/System/Environment.cs

```diff
@@ -543,31 +485,14 @@ private unsafe static char[] GetEnvironmentCharArray()
             }
         }
 
-        
-        /*==================================WorkingSet==================================
-        **Action:
-        **Returns:
-        **Arguments:
-        **Exceptions:
-        ==============================================================================*/
-        [DllImport(JitHelpers.QCall, CharSet = CharSet.Unicode), SuppressUnmanagedCodeSecurity]
-        private static extern long GetWorkingSet();
```

Comment:
Removed. I think there are other things in ecalllist.h that are dead but I didn't go through.